### ISSUE TITLE
Confusing project path on windows bar

### ISSUE THREAD

**Issue by StatelessCat** [atom/atom#7728]
Hey,
When more than one projects are opened on the editor, if we edit a file on the project added after the first one, the path shown just after the name of the opened file in the windows title bar is always the path of the first opened project.
This can cause confusion for the user if the same file name is present in multiple projects.

Step to reproduce:

* Add projectA, and projectB.
* Open a file on projectB.
* Look at the windows title bar :
`file - /home/statelesscat/projetAPath - Atom`

**Comment by benogle** [comment]
cc @maxbrunsfeld Might be nice to show the project that the active file comes from.

**Comment by 50Wliu** [comment]
I can't reproduce this on Windows 7, Atom 1.0.0.  Switching files between multiple projects updates the titlebar correctly.

**Comment by chuandadexiaoyu** [comment]
on centos7, I can't reproduce this, which platform are you using ?

**Comment by StatelessCat** [comment]
After trying to reproduce it this morning (Archlinux, KDE Plasma 5), the behavior I described no longer exists. I think this issue can be closed because not Atom related.

Maybe this was caused by my windows manager that restart itself sometimes and maybe something was not communicate well after that.
I know nothing about communication between windows and windows-manager, does the title is send to the windowsmanager in one chunk?
